packages/php-db-import-export/tests/functional/Snowflake/ToFinal/IncrementalImportTest.php

```diff
@@ -106,19 +109,65 @@ public function testLoadTypedTableWithCastingValues(): void
         );
         /** @var SnowflakeTableDefinition $destination */
         $destination = $destinationRef->getTableDefinition();
-        $stagingTable = StageTableDefinitionFactory::createVarcharStagingTableDefinition(
-            $destination->getSchemaName(),
-            [
+        $columnCollection = new ColumnCollection([
```

Comment:
connection uz vyrabi typovane stage tabulky, takze se musi i tady

Reply:
Jakože teoreticky to ta lib dovoluje pořád. Už bych ju radši zrušil :D

Reply:
jo, ale chceme to testovat? nebo asi jo, ale primarne chci videt case s tim co od te libky chceme

Reply:
jj asi není potřeba mět pokrytů každů kravinu